README.md

```diff
@@ -1,12 +1,16 @@
 
 # Alfresco Content Services Community Deployment
 
-
 ### Contributing guide
 Please use [this guide](CONTRIBUTING.md) to make a contribution to the project.
 
 This project contains the code for starting the entire Alfresco Content Services (Community Edition) product with **Docker** or **Kubernetes**.
 
+The **master** branch of this repository will endeavour to support the following deployments:
+- [Docker Compose](docs/docker-compose-deployment.md) (latest): For development and trials
+- [MiniKube](docs/helm-deployment-minikube.md) (latest): For development and POCs
+- [Helm - AWS Cloud with Kubernetes](docs/helm-deployment-aws_cloud.md) (latest): For production or as a basis of production deployments
```

Comment:
@amohammedalfresco Please link to the acs-deployment project in lines 10 - 12 (similar to README.md)